Description: Do not access network when building docs
 The upstream conf.py is using sphinx.ext.intersphinx, this patch removes it.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2014-01-09

--- python-logutils-0.3.3.orig/doc/conf.py
+++ python-logutils-0.3.3/doc/conf.py
@@ -26,7 +26,7 @@ sys.path.append(os.path.abspath('..'))
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.intersphinx']
+extensions = ['sphinx.ext.autodoc']
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
